lm75bd/lm75bd.c

```diff
@@ -26,7 +26,39 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
+  error_code_t errCode;
+  
+  if (temp == NULL) {
+    return ERR_CODE_INVALID_ARG; 
+  }
+  
+  uint8_t temp1 = TEMP_ADDR;
```

Comment:
This isn't very descriptive. This variable holds the address of the temperature register. So something like `tempRegAddr` would be better.